docs/SOPs/HCA_Metadata_Spreadsheet_Guide_Wranglers.md

```diff
@@ -313,23 +303,23 @@ The highest level biological entity (often Donor) is not ‘derived from’ anyt
 
 One process entity is automatically created between every biomaterial, protocol, and file entity in the graph. A process entity contains metadata about the transition between entities. The process itself contains the who, when, where metadata which is linked to protocol entities with detailed metadata about the methods. One process can link to zero, one, or more protocols.
 
-![image alt text](../../images/ssheet_guidance_image_3.png)
+![image alt text](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/spreadsheet_guidance/ssheet_guidance_image_3.png)
 
 ### Protocols
 
 Protocols need to be linked to the processes sandwiched between biomaterial and/or file entities.
 
-![image alt text](../../images/ssheet_guidance_image_4.png)
+![image alt text](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/spreadsheet_guidance/ssheet_guidance_image_4.png)
 
 To create a link from one collection protocol to 5 processes between ‘donor organism’ and ‘specimen from organism’, we add the collection protocol ID to the specimen tab. **We always add protocol IDs to the lower entity in the hierarchy**.
 
-![image alt text](../../images/ssheet_guidance_image_5.png)
+![image alt text](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/spreadsheet_guidance/ssheet_guidance_image_5.png)
 
 Although this sample shows the linking of one protocol, multiple protocols can be added to a process.
 
 ### Bundles
```

Comment:
```suggestion
### Subgraphs
```

Reply:
I forgot to commit this change as well, but I've seen that the word bundle is still in a number of places.
What is the difference between bundle and subgraph? Do we want to change all occurrences of bundle?

Reply:
it's not super important but we should be consistent

Basically, there is no difference in the terms but that's how we used to call the subgraphs. There were several discussions and now we call them subgraphs instead of bundles, so our documentation should reflect that